packages/dd-trace/src/encode/agentless-ci-visibility.js

```diff
@@ -1,18 +1,16 @@
 'use strict'
 const { truncateSpan, normalizeSpan } = require('./tags-processors')
-const Chunk = require('./chunk')
 const { AgentEncoder } = require('./0.4')
 const { version: ddTraceVersion } = require('../../../../package.json')
 const id = require('../../../dd-trace/src/id')
-
 const ENCODING_VERSION = 1
 
 const ALLOWED_CONTENT_TYPES = ['test_session_end', 'test_suite_end', 'test']
 
 const TEST_SUITE_KEYS_LENGTH = 11
 const TEST_SESSION_KEYS_LENGTH = 10
 
-const CHUNK_SIZE = 4 * 1024 * 1024 // 4MB
+const INTAKE_SOFT_LIMIT = 2 * 1024 * 1024 // 2MB
```

Comment:
when it reaches the limit, the intake can return a HTTP 413 Payload Too Large

Reply:
that's not the chunk size but the limit at which we flush. Bigger payloads are rejected by the agentless intake

Reply:
Ah yes sorry I misread that since it replaces the old variable.